tsconfig.json

```diff
@@ -1,13 +1,12 @@
 {
   "compilerOptions": {
-    "baseUrl": ".",
-    "esModuleInterop": true,
-    "inlineSources": true,
-    "module": "commonjs",
-    "moduleResolution": "node",
-    "sourceMap": true,
-    "strict": true,
-    "target": "es6"
+    "noEmit": true
   },
-  "include": ["./types/**/*.d.ts", "./src/**/*.ts"]
+  "references": [
```

Comment:
Here we're indicating that in order to build the whole project, TypeScript must consider these subprojects.